dash-renderer/package.json

```diff
@@ -9,9 +9,10 @@
     "prepublish": "npm test && npm run build && npm run build:dev",
     "publish-all": "npm publish && echo \"twine!!!\"",
     "publish-pypi": "npm run prepublish && python setup.py sdist upload",
-    "build:dev": "webpack --mode development",
-    "build:dev.watch": "webpack --mode development --watch",
-    "build": "webpack --mode production",
+    "build:renderer": "webpack --mode development && webpack --mode production",
+    "build:dev": "webpack --mode development --watch",
+    "build": "renderer npm && renderer bundles",
+    "build:ci": "renderer npm && renderer bundles True",
```

Comment:
might remove this after testing, the bunles with `True` will use the prod react bundles for all like before

Reply:
If I understand this well, shouldn't `build` use `True` and `build:dev` use `renderer npm && renderer bundles`?

Reply:
I will remove this option in the code, as it turns out that it's not an issue with the dev version.